src/common/base-chart.component.ts

```diff
@@ -23,6 +23,7 @@ export class BaseChartComponent implements OnChanges, AfterViewInit, OnDestroy {
 
   @Output() select = new EventEmitter();
 
+  designatedTotal: number;
```

Comment:
This doesn't look like it belongs to the base chart, as it it not used by all charts of the library. Perhaps move it to the number card and pie grid components?

Reply:
Ok, good point; I've moved to the number card component.